changelog/unreleased/bugfixes/6986.md

```diff
@@ -0,0 +1,9 @@
+- Improved duration calculation for EV routes to account for charging time.
+- Added snapping feature to the navigation for walking and cycling profiles.
+- Started invalidating passed alternatives conditionally on the fork structure.
+- Started using offline router if server returns 404 or 403.
+- Delayed discarding alternatives until MapMatching algorithm is confident enough.
```

Comment:
```suggestion
- Increased the distance at which the navigator discards passed alternative route in `NavigationRouteAlternativesObserver#onRouteAlternatives`. This reduces the chance of discarding alternative routes to which a driver deviated from the primary route.
```
I have already described this feature in this way here: https://github.com/mapbox/mapbox-navigation-android/pull/6966/files

But maybe it's okay to have different descriptions for different versions if they're both great 🙂